src/modules/Dashboard/features/Participants/Participants.utils.tsx

```diff
@@ -30,15 +80,76 @@ export const getParticipantActions = ({
   tag,
   status,
   dataTestid,
+  invitation,
   showAssignActivity = false,
-}: GetParticipantActionsProps) => {
-  const context = { respondentId, respondentOrSubjectId, email, secretId, nickname, tag };
+}: GetParticipantActionsProps): MenuItem<ParticipantActionProps>[] => {
+  const context = {
+    respondentId,
+    respondentOrSubjectId,
+    email,
+    secretId,
+    nickname,
+    tag,
+    invitation,
+  };
   const isUpgradeable = status === RespondentStatus.NotInvited;
   const isPending = status === RespondentStatus.Pending;
   const isEditable = !!filteredApplets?.editable.length;
   const isViewable = !!filteredApplets?.viewable.length;
 
-  return [
+  let title = '';
+  const emailAddress = email || invitation?.email;
+
+  if (emailAddress) {
+    title += `${i18n.t('email')}: ${emailAddress}`;
+  }
+
+  const hasInvitation = !!invitation;
+
+  if (hasInvitation) {
+    if (invitation.firstName && invitation.lastName) {
+      const fullName = `${i18n.t('fullName')}: ${invitation.firstName} ${invitation.lastName}`;
+      title += title.length > 0 ? '\n' : '';
+      title += fullName;
+    }
+
+    const invitationDate = `${i18n.t('invitationDate')}: ${format(
+      new Date(invitation.createdAt),
+      DateFormats.MonthDayYearTime,
+    )}`;
+
+    title += title.length > 0 ? '\n' : '';
+    title += `${invitationDate}`;
+  }
+
+  const hasTitle = title.length > 0;
+
+  return cleanUpDividers([
+    { title, isDisplayed: hasTitle },
```

Comment:
This item has hover state and `cursor: pointer`, is this expected?

![CleanShot 2024-05-31 at 10 04 45](https://github.com/ChildMindInstitute/mindlogger-admin/assets/2916156/1a664945-c48a-4446-a12c-14134a00edf4)

Reply:
I don't think it's meant to have these, but there's no special header item in the action menu so it just inherits the same behaviour as actual actions. I'll see to removing it